### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by vv01f**
* convert json to csv
* describe data source in a config file, so it may work for other arcgis sources as well
* PR for schedule action

### CODE REVIEW

File: .github/workflows/data-de_dresden_www.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  push:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+     
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```
Comment (psteinb) [inline .github/workflows/data-de_dresden_www.yml:17]: Maybe a second order optimization: if this is the name of a docket image, using alpine and friends could speed up things.

**Comment by psteinb** [comment]
That is quite some shell code. The gh actions code is intriguing.

**Comment by vv01f** [comment]
I am still a bit clumsy with github… but [the environments available are documented](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources) and as of now I do not see how to use an alpine linux although I think using sth else via e.g. docker hub should be possible.

**Comment by vv01f** [comment]
now I am just curious if the cron trigger works as expected and my repo has a new line in that file tomorrow wee past noon.

**Comment by psteinb** [comment]
let me know, when you think this is good for merging.

**Comment by vv01f** [comment]
As we now know that (and when is UTC) scheduling works I think it's fine to merge.
to make it winter-proof 11:10 (or running twice and checking in there) would be a better idea, but let's hope we wont need that.
maybe comment/drop whatever step/information you do not find helpful.

**Comment by psteinb** [comment]
I'll merge now. I hope this dumps the data in the right location. We'll see about that. Feel free to change the time to 11:10. As long as we are aware of this, it's fine.

**Pull request merged**